src/main/java/io/stargate/sgv2/jsonapi/config/constants/CommonConstants.java

```diff
@@ -0,0 +1,13 @@
+package io.stargate.sgv2.jsonapi.config.constants;
+
+/** Contains common constants shared in different parts of the codebase. */
+public interface CommonConstants {
```

Comment:
This seems like a very generic name for something that is still specific to providers (vector/embedding, reranking). Not sure what'd be better, or if there is even need for such containe, just pointing out a small concern.